skills/lark-im/references/card/components/checker.md

```
@@ -0,0 +1,38 @@
+```
+
+## 字段
+
+| 字段 | 必填 | 类型 | 默认 | 说明 |
+|---|---|---|---|---|
+| `tag` | 是 | String | / | 固定 `checker` |
+| `name` | 否* | String | / | 唯一标识；**form 内必填且全局唯一** |
+| `checked` | 否 | Boolean | false | 初始勾选状态 |
+| `text` | 否 | Object | / | `{tag:"plain_text"\|"lark_md", content, text_size?, text_color?, text_align?}`（text_color 见 `../resource/colors.md`） |
+| `overall_checkable` | 否 | Boolean | true | 悬浮时整体是否有阴影效果 |
+| `button_area` | 否 | Object | / | `{pc_display_rule:"always"|"on_hover", buttons:[<=3 个 button]}` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 5; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)